CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 Records breaking changes from major version bumps
 
+## 28.0.0
+
+PR: [#397](https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/397)
+
+### What changed
+
+- Changes existing header to product page style header.
+- Product page Github: https://github.com/alphagov/product-page-example
```

Comment:
Could we have in here what changes someone would need to make to a frontend app to bring this new version of the toolkit in? As in, I assume if one of the FE apps just bumped to this toolkit version without any other changes then things would break/look wrong?

E.g. mention the changes that you are currently having to make in the buyer FE.
For example does the FE need a certain version of utils pulled in?
Removal of phase banner
Adding of `inside-header` block
That type of stuff

Reply:
I have updated the instructions below